cloudmarker/test/data/docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+# Use root/example as user/password credentials
+version: '3.1'
+
+services:
+
+  mongo:
+    image: mongo
+    restart: always
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: dummy
+      MONGO_INITDB_ROOT_PASSWORD: password
+    ports:
+      - 27017:27017
+    networks:
+      mynet:
+        ipv4_address: 172.16.101.2
+
+  mongo-express:
+    image: mongo-express
+    restart: always
+    ports:
+      - 8081:8081
+    environment:
+      ME_CONFIG_MONGODB_ADMINUSERNAME: dummy
+      ME_CONFIG_MONGODB_ADMINPASSWORD: password
+    networks:
+      mynet:
+        ipv4_address: 172.16.101.3
+
+networks:
```

Comment:
Why are you choosing a specific subnet for the containers? what I feel is like, we should let the docker spin up the network that it wants to and attach the containers to it. I could not see any use of adding network here. 
Is it that networks are mandatory in compose version `3.1` and if it so do we have a simpler way of referencing the container instead of static IPs.